ui/app/src/components/CreateSiteDialog/GitForm.tsx

```diff
@@ -176,7 +176,12 @@ function GitForm(props: GitFormProps) {
       </Grid>
       <Grid item xs={12}>
         <div className={classes.formControl}>
-          <GitAuthForm inputs={inputs} setInputs={setInputs} handleInputChange={handleInputChange} />
+          <GitAuthForm
+            inputs={inputs}
+            setInputs={setInputs}
+            handleInputChange={handleInputChange}
+            onKeyPress={onKeyPress}
```

Comment:
Check the rest of the dialog for same issue

Reply:
The only field that doesn't have it now is the description field, because it is a multi line text field so handling the enter would cause unwanted behavior